sm_value_store/include/sm/value_store/ValueStore.hpp

```diff
@@ -221,7 +221,7 @@ class KeyValueStorePair : public ValueStoreRef {
 };
 
 class ExtendibleKeyValueStorePair;
-class ExtendibleValueStore : public ValueStore {
```

Comment:
I dont see how this would work
If I use explicit instatiation of Base for impl on ValueStore, how would then extendibleVerboseValueStore have them overridden the same way, as its base is VerboseValueStoreT<ExtendibleValueStore> so the explicit instatiation does not affect it no?

Reply:
Th explicit instantiation would need to be effectively for  `internal::VerboseValueStoreT<ValueStore>` and `internal::VerboseValueStoreT<ExtendibleValueStore>`. This way their methods would not need to be _defined_ (only declared) in the header.